pkg/cmd/pkg/package_repo_list.go

```diff
@@ -18,7 +26,37 @@ func newCmdPackageRepoList(ctx api.Context) *cobra.Command {
 		Long:  description,
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+
+			repos, err := c.PackageListRepo()
+			if err != nil {
+				return err
+			}
+
+			if jsonOutput {
+				enc := json.NewEncoder(ctx.Out())
+				enc.SetIndent("", "    ")
+				return enc.Encode(repos)
+			}
+
+			// TODO: this follows the python behavior but is this really an error?
```

Comment:
I'm pretty sure we had this case already while refactoring but don't remember about our decision, I'll try to search tomorrow on which `list` command it was. @rgo3 any suggestion?

Reply:
Looking at other commands like `dcos task list`, `dcos job list`, `dcos cluster list`, etc. We don't fail when there is no entries, but rather only print the table header. We should do the same for consistency.